### DISCUSSION TITLE
Remove top shadow from android

### DISCUSSION THREAD

**Discussion by abdallahkadour** [NativeScript/NativeScript#10248] [General]
Any idea how to remove the shadow as shown in the image?
Thanks in advance!

![FA6EC676-E543-4785-8315-1E76AB518551](https://user-images.githubusercontent.com/12232324/227464936-e4d14652-531b-4644-905d-4d34a181ba00.jpeg)

**Comment by abdallahkadour** [comment]
I defined an activity which extends `androidx.appcompat.app.AppCompatActivity`
and tried `this.getSupportActionBar().setElevation(0)` but the  getSupportActionBar() is always undefined

**Reply by rigor789** [reply]
That seems to be the `StatusBar`, you can change it's color: https://docs.nativescript.org/app-resources.html#setting-the-default-color-of-the-status-bar

Though not sure if that works, as it seems like you already have `backgroundSpanUnderStatusBar` enabled? Maybe you'd want to disable it and change the color like above...

**Comment by abdallahkadour** [comment]
@rigor789 Thanks for your reply. As you said changing the color solves the problem. I did it as described here https://github.com/NathanWalker/nativescript-flavorful/blob/main/apps/angular/src/utils/status-bar.ts

**Reply by abdallahkadour** [reply]
and disabled `backgroundSpanUnderStatusBar `
